Reconstruction/SiliconTracking/src/SiliconTrackingAlg.cpp

```diff
@@ -2793,11 +2864,13 @@ void SiliconTrackingAlg::FinalRefit(edm4hep::TrackCollection* trk_col) {
 	error() << "Cannot create MarlinTrack ! " << endmsg;
 	return;
       }
+      */
```

Comment:
Hi @fucd, 
Is this part not used any more? If it is not used, please consdier to remove them.
Tao

Reply:
Yes, I have removed them.